llama-index-integrations/llms/llama-index-llms-anthropic/pyproject.toml

```diff
@@ -27,7 +27,7 @@ exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-llms-anthropic"
 readme = "README.md"
-version = "0.3.4"
+version = "0.3.5"
```

Comment:
We actually don't need to vbump these for README updates -- is it easy to undo these? 😅

Reply:
I think yeah for llamahub we don't need to update the version but for PYPI to reflect the new readme update we do.